src/content/blog/2024-10-22-rafikis-first-security-audit.mdx

```diff
@@ -0,0 +1,51 @@
+---
+layout: ../../layouts/BlogLayout.astro
+title: "Rafiki's First Security Audit"
+description: "Takeaways from Rafiki's 2023 security audit"
+date: 2024-10-22
+slug: rafikis-first-security-audit
+authors: [Max Kurapov]
+author_urls: [https://www.linkedin.com/in/mkurapov]
+tags:
+  - Rafiki
+  - security
+  - audit
+---
+
+At the beginning of the year, we were in contact with a security and penetration testing company to do an audit of Rafiki. Even though the software is still in its early stages, it is essential to gather feedback early to build a strong foundation for the software's security. The primary goals of the assessment were to evaluate several Rafiki components: the GraphQL Admin APIs, the frontend Admin UI component, as well as our underlying ILPv4 protocol.
+
+The assessment was done using Rafiki’s local playground, based on the Open Source Security Testing Methodology Manual (OSSTMM) and Open Source Web Application Security Project (OWASP) methodologies. Given all of the source code for Rafiki and the local playground is readily available, this was deemed as a “crystal-box” test, meaning all of the source code, endpoints, architecture, etc. was known in-advance to the testers. 
```

Comment:
maybe add links for OSSTMM and OWASP?

Reply:
edited the section a bit, removed reference to crystal box since it didn't really involve the testers going deep into the code and testing branching logic etc